test/eventHandlers/workerInitRequest.test.ts

```diff
@@ -11,6 +12,44 @@ import { TestEventStream } from './TestEventStream';
 import LogCategory = rpc.RpcLog.RpcLogCategory;
 import LogLevel = rpc.RpcLog.Level;
 
+namespace Msg {
+    export function initMsg(functionAppDirectory?: string): rpc.IStreamingMessage {

```

Comment:
nit: You don't need "Msg" at the end of these names when they're in a "Msg" namespace